modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/ExportImportHelperImpl.java

```diff
@@ -1402,6 +1357,53 @@ else if (rootPortletId != null) {
 		return importPortletSetupControlsMap;
 	}
 
+	private List<Portlet> _getPortlets(
+		long companyId, DataLevel[] dataLevels,
+		boolean excludeDataAlwaysStaged) {
+
+		List<Portlet> portlets = new ArrayList<>();
+
+		Map<Integer, List<Portlet>> rankedPortletsMap = new TreeMap<>();
+
+		for (Portlet portlet : _portletLocalService.getPortlets(companyId)) {
+			if (!portlet.isActive()) {
+				continue;
+			}
+
+			PortletDataHandler portletDataHandler =
+				portlet.getPortletDataHandlerInstance();
+
+			if ((portletDataHandler == null) ||
+				!ArrayUtil.contains(
+					dataLevels, portletDataHandler.getDataLevel()) ||
+				((portletDataHandler.isDataPortalLevel() ||
+				  portletDataHandler.isDataSiteLevel()) &&
+				 !portletDataHandler.isEmptyControlsAllowed()) ||
```

Comment:
We check here whether `isEmptyControlsAllowed`, but not the actual asset count of the portlet data. That's why old asset types, like JournalArticles or BlogEntries are not displayed on export and staging forms anymore.